user_docs/en/userGuide.t2t

```diff
@@ -2072,13 +2072,16 @@ Please see the display's documentation for descriptions of where these keys can
 | Say all | B6 |
 %kc:endInclude
 
-++ HumanWare Brailliant BI/B Series ++[HumanWareBrailliant]
-The Brailliant BI and B series of displays from [HumanWare http://www.humanware.com/], including the BI 32, BI 40 and B 80, are supported when connected via USB or bluetooth.
+++ HumanWare Brailliant BI/B Series, and BrailleNote Touch  ++[HumanWareBrailliant]
+The Brailliant BI and B series of displays  from [HumanWare http://www.humanware.com/], including the BI 14, BI 32, BI 40 and B 80, are supported when connected via USB or bluetooth.
 If connecting via USB with the protocol set to HumanWare, you must first install the USB drivers provided by the manufacturer.
 USB drivers are not required if the protocol is set to OpenBraille.
 
-Following are the key assignments for this display with NVDA.
+The BrailleNote Touch is also supported, and does not require any drivers to be installed.
+
+Following are the key assignments for  the Brailliant BI/B, and BrailleNote touch displays with NVDA.

```

Comment:
Note the comma here as well. Now I'm not a native English speaker, but I belief a comma before `and` is only prescribed in case three or more entities are involved (i.e. one, two, and three).